### DISCUSSION TITLE
3.20默认不开放https的5245端口吗？

### DISCUSSION THREAD

**Discussion by Masterlovsky** [AlistGo/alist#4699] [Q&A]
更新之后5245端口的url失效了，检查了一下配置文件发现变成了-1

**Comment by xhofe** [comment]
原本默认也是不启用的 现在改了位置 需要自己修改一下。
原来是两个设置 https端口和是否开启https，现在是只有端口 -1表示不启用 其他启用。

**Reply by lewis-yeung** [reply]
[文档](https://alist.nn.ci/zh/config/configuration.html)也需要更新。

**Reply by AspirantH** [reply]
请问在哪修改，/etc/alist下的配置文件更改https端口不生效
  "scheme": {
    "address": "0.0.0.0",
    "http_port": 5244,
    "https_port": 8888,
    "force_https": false,
    "cert_file": "/etc/uhttpd.crt",
    "key_file": "/etc/uhttpd.key",
    "unix_file": ""
  },

**Reply by xhofe** [reply]
目前有个bug我将在稍后修复它

**Reply by AspirantH** [reply]
辛苦辛苦，大佬回复速度感人～
